libraries/stumbler/src/main/java/org/mozilla/mozstumbler/service/stumblerthread/datahandling/PersistedStats.java

```diff
@@ -111,14 +131,32 @@ private void sendToListeners(Properties newProps) {
     }
 
     private synchronized Properties writeSyncStats(long time, long bytesSent, long totalObs,
-                                            long totalCells, long totalWifis, long obsPerDay) throws IOException {
-        final FileOutputStream out = new FileOutputStream(mStatsFile);
+                                            long totalCells, long totalWifis, long obsPerDay) {
+        final FileOutputStream out;
```

Comment:
should this be out = null;
is this defaulted to null? I know class members are, not sure about fun-local vars.